var/spack/repos/builtin/packages/tbl2asn/package.py

```diff
@@ -14,12 +14,12 @@ class Tbl2asn(Package):
 
     homepage = "https://www.ncbi.nlm.nih.gov/genbank/tbl2asn2/"
 
-    version('2020-03-01', sha256='7cc1119d3cfcbbffdbd4ecf33cef8bbdd44fc5625c72976bee08b1157625377e')
+    version('2020-03-01', sha256='b7edf378c971e6d7c48bf4d391d43a6d79f0e26dfa57156b4f87456af83e1fee')
 
     def url_for_version(self, ver):
-        return "https://ftp.ncbi.nih.gov/toolbox/ncbi_tools/converters/by_program/tbl2asn/linux.tbl2asn.gz"
+        return "https://ftp.ncbi.nih.gov/toolbox/ncbi_tools/converters/by_program/tbl2asn/linux64.tbl2asn.gz"
 
     def install(self, spec, prefix):
         mkdirp(prefix.bin)
-        install('../linux.tbl2asn', prefix.bin.tbl2asn)
-        chmod(prefix.bin.tbl2asn, 0o775)
+        install('../linux64.tbl2asn', prefix.bin + '/tbl2asn')
```

Comment:
Just in case the old version resurfaces, make this conditional
```suggestion
        if spec.version >= Version('2021'):
            install('../linux64.tbl2asn', prefix.bin + '/tbl2asn')
        else:
            install('../linux.tbl2asn', prefix.bin + '/tbl2asn')
```

Reply:
tbl2asn only runs when it is within one year of its release date; there is no reason why it would resurface.